src/backend/bot.py

```diff
@@ -113,17 +114,20 @@ async def on_message_activity(self, turn_context: TurnContext):
                               max_tokens=1500, callback_manager=cb_manager, streaming=True)
 
         # Initialize our Tools/Experts
-        doc_indexes = ["cogsrch-index-files", "cogsrch-index-csv"]
+        # doc_indexes = ["cogsrch-index-files", "cogsrch-index-csv"]
```

Comment:
I would remove these things rather than comment them out.  But the refactoring I'm working on will take care of that, so don't worry.